packages/itinerary-body/src/ItineraryBody/accessibility-rating.tsx

```diff
@@ -74,11 +77,12 @@ const AccessibilityRating = ({
 
   return (
     <Wrapper
+      border={grayscale}
+      color={grayscale ? "transparent" : mapped.color}
       large={large}
-      color={grayscale ? "#eee" : mapped.color}
       title={mapped.text}
     >
-      <Wheelchair style={{ flex: "2", minWidth: "20px" }} />
+      <Wheelchair style={{ flex: "2", minWidth: "20px", height: "100%" }} />
```

Comment:
Sort fields.

Reply:
Sorted in d8db56ebd3f28f69acd09554ca296d72ff3a3ee6